src/main/java/step3_racing_car/MovingStrategy.java

```diff
@@ -0,0 +1,5 @@
+package step3_racing_car;
+
+interface MovingStrategy {
+    int getMovingDistance();
+}
```

Comment:
크게 중요한 부분은 아닙니다만, `@FunctionalInterface` 을 붙여 람다 형태로 쓸 수 있다는 정보를 주어도 괜찮습니다.

메소드명도 move() 정도로 축약해도 크게 나쁘진 않겠네요.

get이나 find는 데이터소스에서 무언가를 가져오는 느낌으로 쓰거든요.


```suggestion
@FunctionalInterface
interface MovingStrategy {
    int move();
}
```